roles/lib_openshift/src/test/unit/test_oc_project.py

```diff
@@ -115,9 +115,8 @@ def test_adding_a_project(self, mock_cmd, mock_write, mock_tmpfile_copy, mock_lo
         mock_cmd.assert_has_calls([
             mock.call(['oc', 'get', 'namespace', 'operations', '-o', 'json'], None),
             mock.call(['oc', 'get', 'namespace', 'operations', '-o', 'json'], None),
-            mock.call(['oc', 'adm', 'new-project', 'operations', '--admin-role=admin',
-                       '--display-name=operations project', '--description=All things operations project',
-                       '--node-selector=ops_only=True'], None),
```

Comment:
@detiber, I like this change.  I think we should create a card and place these types of enhancements there.